### PR TITLE
add some link

### PR DESCRIPTION
**Pull request by liberhe**

### CODE REVIEW

File: basic/48-web3-rust/readme.md [hunk 1]
```diff
@@ -71,6 +71,7 @@ cargo run
 
 ## 参考文档
 
+- rust语言圣经： https://course.rs/basic/variable.html
```
Comment (0x-stan) [inline basic/48-web3-rust/readme.md:74]: https://course.rs

File: basic/58-EVM/libary.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+3. 使用代理；
+一个更先进的策略是代理系统。 库在后台使用 DELEGATECALL，它只是用调用合约的状态执行另一个合约的函数；
+
+中等影响：
+1. 在优化器中考虑一个低运行值：
+您也可以更改优化器设置。 默认值为 200，表示它试图在一个函数被调用 200 次的情况下优化字节码。 如果您将其改为 1，相当于告诉优化器针对每个函数只运行一次的情况进行优化。 一个仅运行一次的优化函数意味着它对部署本身进行了优化。 请注意，这将会增加运行函数的 gas 成本，所以，您可能不想这样做。
+
+
+##  Diamond Standard（EIP-2535）
+https://dev.to/mudgen/ethereum-s-maximum-contract-size-limit-is-solved-with-the-diamond-standard-2189
```
Comment (0x-stan) [inline basic/58-EVM/libary.md:34]: 404

**Review by 0x-stan — DISMISSED** [review]

File: basic/58-EVM/libary.md (hunk 2 above)
Comment (liberhe) [reply basic/58-EVM/libary.md:34]: it works

File: basic/48-web3-rust/readme.md (hunk 1 above)
Comment (liberhe) [reply basic/48-web3-rust/readme.md]: done

**Review by 0x-stan — APPROVED** [review]

**Review by yingjingyang — APPROVED** [review]

**Pull request merged**